fuel-asm/src/args.rs

```diff
@@ -248,6 +248,9 @@ crate::enum_try_from! {
 
         /// Set `$rA` to `tx.policies[count_ones(0b11111 & tx.policyTypes) - 1].expiration`
         PolicyExpiration = 0x505,
+
+        /// Set `$rA` to `tx.policies[count_ones(0b111111 & tx.policyTypes) - 1].owner`
```

Comment:
this applies also to comments for other variants, but I could not find `tx.policyTypes` anywhere, so not sure how I should interpret `0b111111 & tx.policyTypes`

Reply:
found it, NVM :)